src/index.js

```diff
@@ -1,3 +1,310 @@
-import { example } from './example.js';
+import { renderLogin } from './views/login.js';
+
+renderLogin();
+
+
+/*const db = firebase.firestore();
```

Comment:
borren lo que no se utiliza, es mucho código comentado
```suggestion
/*const db = firebase.firestore();
```